spec/controllers/password_resets_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe PasswordResetsController do
+  describe "GET show" do
+    it "renders show template if the token is valid" do
+      alice = Fabricate(:user)
+      alice.update_column(:token, '12345')
+      get :show, id: '12345'
+      expect(response).to render_template :show
+    end
+
+    it "sets @token" do
+      alice = Fabricate(:user)
+      alice.update_column(:token, '12345')
+      get :show, id: '12345'
+      expect(assigns(:token)).to eq('12345')
+    end
+
+    it "redirects to the expired token page if the token is not valid" do
+      get :show, id: '12345'
+      expect(response).to redirect_to expired_token_path
+    end 
+  end
+
+  describe "POST create" do
+    context "with valid token" do
+
+      it "redirect to the sign page" do
+        alice = Fabricate(:user, password: 'old_password')
+        alice.update_column(:token, '12345')
+        post :create, token: '12345', password: 'new_password'
+        expect(response).to redirect_to sign_in_path 
+      end
+
+      it "updates the user's password" do
+        alice = Fabricate(:user, password: 'old_password')
+        alice.update_column(:token, '12345')
+        post :create, token: '12345', password: 'new_password'
+        expect(alice.reload.authenticate('new_password')).to be_true
```

Comment:
`be_true` has been [deprecated](https://github.com/rspec/rspec-expectations#truthiness). The correct syntax here is now `be_truthy`